modules/drivers/mongo/src/metabase/driver/mongo.clj

```diff
@@ -55,10 +55,12 @@
 (defmethod driver/can-connect? :mongo
   [_ details]
   (with-mongo-connection [^DB conn, details]
-    (= (float (-> (cmd/db-stats conn)
-                  (m.conversion/from-db-object :keywordize)
-                  :ok))
-       1.0)))
+    (let [db-stats (-> (cmd/db-stats conn)
```

Comment:
what is the reason behind this change?

Reply:
I've just added that explanation as comments
https://github.com/metabase/metabase/blob/5dfee97b97e3234d6f605515a88b529716bd058a/modules/drivers/mongo/src/metabase/driver/mongo.clj#L61-L67